### PR TITLE
Enhanced Cubed Sphere Visualization

### PR DESCRIPTION
**Pull request by siddharthabishnu**
This pull request enhances the visualization of the cubed sphere by adjusting the transparency of grid lines on panels outside the user's field of view.  As a result, the plots achieve a cleaner and more aesthetically pleasing appearance, thereby enhancing the overall user experience.

Below are example plots of the cubed sphere and its 2D projection before and after the modifications introduced in this pull request.

**Before modification:**

cubed_sphere_2D_3D_Before_Modification

**After modification:**

cubed_sphere_2D_3D_After_Modification

### CODE REVIEW

**Review by navidcy — CHANGES_REQUESTED** [review]
@siddharthabishnu I don't see why you added this in `test` directory since they are not really tests (they don't return true/false). I think you want to modify the docs and the README, right?

**Comment by siddharthabishnu** [comment]
> @siddharthabishnu I don't see why you added this in `test` directory since they are not tests (e.g., they don't return true/false). I think you want to modify the docs and the README, right?

@navidcy, yes, you are right. I will fix it in the next commit.

File: docs/src/conformal_cubed_sphere.md [hunk 1]
```diff
@@ -54,43 +64,44 @@ colgap!(fig.layout, 40)
...
 
-ax2D = Axis(fig[1, 1], aspect = 1)
-ax3D = Axis3(fig[1, 2], aspect = (1, 1, 1), limits = ((-1, 1), (-1, 1), (-1, 1)))
+ax2D = Axis(fig[1, 1]; aspect = 1, title = "Cubed Sphere", axis_kwargs_2D...)
+ax3D = Axis3(fig[1, 2]; aspect = (1, 1, 1), limits = ((-1, 1), (-1, 1), (-1, 1)), title = "Cubed Sphere", 
+             axis_kwargs_3D...)
 
 for ax in [ax2D, ax3D]
-    wireframe!(ax, X, Y, Z)
+    wireframe!(ax, X, Y, Z, color = colors[1], alpha = alphas[1])
```
Comment (navidcy) [inline docs/src/conformal_cubed_sphere.md:81]: this line errors..

https://github.com/CliMA/CubedSphere.jl/actions/runs/7821625824/job/21338990762?pr=34#step:6:403

you can try to build the docs locally by calling:

```
rm -rf docs/build; JULIA_DEBUG=Documenter julia --color=yes --project=docs docs/make.jl
```

and then open `docs/build/index.html`.

File: docs/src/conformal_cubed_sphere.md [hunk 2]
```diff
@@ -38,14 +38,24 @@ for (j, y′) in enumerate(y), (i, x′) in enumerate(x)
...
-fig = Figure(resolution = (800, 400))
+axis_kwargs_2D = (xlabelsize = 22.5, ylabelsize = 22.5, xticklabelsize = 17.5, yticklabelsize = 17.5, 
+                  xticklabelpad = 10, yticklabelpad = 10, titlesize = 27.5, titlegap = 15, titlefont = :bold, 
+                  xlabel = "x", ylabel = "y")
 
-ax2D = Axis(fig[1, 1], aspect = 1)
-ax3D = Axis3(fig[1, 2], aspect = (1, 1, 1), limits = ((-1, 1), (-1, 1), (-1, 1)))
+axis_kwargs_3D = (xlabelsize = 22.5, ylabelsize = 22.5, zlabelsize = 22.5, xticklabelsize = 17.5, yticklabelsize = 17.5,
+                  zticklabelsize = 17.5, xticklabelpad = 10, yticklabelpad = 10, zticklabelpad = 10, titlesize = 27.5,
+                  titlegap = 15, titlefont = :bold, xlabel = "x", ylabel = "y", zlabel = "z")
```
Comment (navidcy) [inline docs/src/conformal_cubed_sphere.md:47]: these axis kwargs are not really needed

File: docs/src/conformal_cubed_sphere.md [hunk 3]
```diff
@@ -54,39 +64,41 @@ colgap!(fig.layout, 40)
...
-We can then use [Rotations.jl](https://github.com/JuliaGeometry/Rotations.jl) to rotate
-the face of the sphere that includes the North Pole and this way obtain all six faces of
-the sphere.
+We can then use [Rotations.jl](https://github.com/JuliaGeometry/Rotations.jl) to rotate the face of the sphere that includes the North Pole and this way obtain all six faces of the sphere.
 
 ```@example 1
 using Rotations
 
-fig = Figure(resolution = (800, 400))
+colors = [:purple, :red, :orange, :cyan, :green, :blue]
```
Comment (navidcy) [inline docs/src/conformal_cubed_sphere.md:74]: prescribing the colors is not really needed right?

(I'm just saying this in an attempt to simplify)

**Review by navidcy — APPROVED** [review]

File: docs/src/conformal_cubed_sphere.md (hunk 1 above)
Comment (siddharthabishnu) [reply docs/src/conformal_cubed_sphere.md]: @navidcy thanks for this information. I didn't know how to build the docs locally.

Comment (navidcy) [reply docs/src/conformal_cubed_sphere.md]: It’s the same process as the GitHub action. 

also see https://clima.github.io/OceananigansDocumentation/stable/contributing/#Documentation

**Pull request merged**